programs/bench.c

```diff
@@ -64,9 +64,9 @@
 
 static const size_t maxMemory = (sizeof(size_t)==4)  ?  (2 GB - 64 MB) : (size_t)(1ULL << ((sizeof(size_t)*8)-31));
 
+/* remove this in the future? */
```

Comment:
or make it programmable ?

This parameter is used when generating a synthetic sample, to benchmark without any source (`zstd -b`).

We could use the same convention as `fullbench` here, by using `-P#` to modify the compressibility.
The option could remain hidden, and therefore not need any external documentation (just code comments).